### PR TITLE
DCOS-15397: ucr integration tests

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds system integration tests for force pull,
bridge network and host volumes.

Closes DCOS-15397

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [x] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: system-tests/services/test-apps.js [hunk 1]
```diff
@@ -1212,5 +1217,422 @@ describe("Services", function() {
...
+          .getFormGroupInputFor("Service ID *")
+          .type(`{selectall}{rightarrow}${serviceName}`);
+        cy.root().getFormGroupInputFor("Container Image").type(container);
+        cy.root().getFormGroupInputFor("Command").type(cmd);
+        // Select ucr runtime
+        selectUniversalRuntime();
+        cy.contains("Force Pull Image On Launch").click();
+
+        //
+        // TODO: Due to a bug in cypress you cannot type values with dots
```
Comment (nLight) [inline system-tests/services/test-apps.js]: I see we're copying this sting over and over. If it's a real TODO let's file an issue at least.
Shall we also remove this comment from here?

File: system-tests/services/test-apps.js [hunk 2]
```diff
@@ -1212,5 +1217,422 @@ describe("Services", function() {
...
+          .root()
+          .getFormGroupInputFor("Service ID *")
+          .type(`{selectall}{rightarrow}${serviceName}`);
+        cy.root().getFormGroupInputFor("Container Image").type(container);
+        cy.root().getFormGroupInputFor("Command").type(cmd);
+        // Select ucr runtime
+        selectUniversalRuntime();
+
+        //
+        // TODO: Due to a bug in cypress you cannot type values with dots
```
Comment (nLight) [inline system-tests/services/test-apps.js]: 🔥

File: system-tests/services/test-apps.js [hunk 3]
```diff
@@ -1212,5 +1217,422 @@ describe("Services", function() {
...
+        // Select Voluimes section
+        cy.root().get(".menu-tabbed-item").contains("Volumes").click();
+
+        // Click "Add Local Volume"
+        /**
+
+         */
+        cy.contains("Add Local Volume").click();
+
+        // Setup HTTP endpoint
```
Comment (nLight) [inline system-tests/services/test-apps.js]: :trollface:

File: system-tests/services/test-apps.js [hunk 4]
```diff
@@ -1212,5 +1217,422 @@ describe("Services", function() {
...
+        cy
+          .root()
+          .getFormGroupInputFor("Service ID *")
+          .type(`{selectall}{rightarrow}${serviceName}`);
+        cy.root().getFormGroupInputFor("Container Image").type(container);
+        // Select ucr runtime
+        selectUniversalRuntime();
+
+        //
+        // TODO: Due to a bug in cypress you cannot type values with dots
```
Comment (nLight) [inline system-tests/services/test-apps.js]: 🔥

**Comment by Poltergeist** [comment]
you are right I shouldn't copy over useless comments. I removed them.

**Comment by Poltergeist** [comment]
retest this please.

**Comment by Poltergeist** [comment]
**mesosphere-ci** run integration tests

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

File: system-tests/services/test-apps.js (hunk 1 above)
Comment (jfurrow) [inline system-tests/services/test-apps.js]: Done!

File: system-tests/services/test-apps.js (hunk 2 above)
Comment (jfurrow) [inline system-tests/services/test-apps.js]: Gone!

File: system-tests/services/test-apps.js (hunk 3 above)
Comment (jfurrow) [inline system-tests/services/test-apps.js]: Fixed!

File: system-tests/services/test-apps.js (hunk 4 above)
Comment (jfurrow) [inline system-tests/services/test-apps.js]: Gone!

**Comment by jfurrow** [comment]
@Poltergeist @nLight I cleaned up a few of the remaining comments — I think this is good to go! 🚢

**Pull request merged**